hostprocess/containerd-logger/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# syntax=docker/dockerfile:1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY program.cs ./
+COPY eventFlowConfig.json ./
+RUN dotnet publish -c Release -r win-x64 -o out
+
+# Build runtime image
+FROM mcr.microsoft.com/windows/nanoserver:1809
+WORKDIR /
+COPY --from=build-env /app/out/containerd-logger.exe .
```

Comment:
```suggestion
COPY --from=build-env /app/out/containerd-logger.exe .
COPY --from=build-env eventFlowConfig.json ./
```

Reply:
I left coping the `eventflowConfig.json` out here since it is copied over via config map.